class-two-factor-core.php

```diff
@@ -93,7 +93,7 @@ class Two_Factor_Core {
 	 * @since 0.1-dev
 	 */
 	public static function add_hooks( $compat ) {
-		add_action( 'init', array( __CLASS__, 'get_providers' ) );
+		add_action( 'init', array( __CLASS__, 'get_providers' ) ); // @phpstan-ignore return.void
```

Comment:
@szepeviktor The default phpstan table report doesn't include the actual error identifiers making it hard to ignore them. Is there an output format that includes them?

Reply:
Yes. Here is an example
https://github.com/TicketSwap/phpstan-error-formatter

Reply:
but there is https://github.com/phpstan/phpstan-src/pull/3112
https://phpstan.org/user-guide/output-format

Reply:
`raw` should also display it

Reply:
Unfortunately, it doesn't:

![no-error-ident](https://github.com/user-attachments/assets/89f75448-c504-4896-abc4-ddd22a29f82a)

Reply:
try adding a `-v`
https://github.com/phpstan/phpstan-src/blob/62c81bac66f2558c2768914d3a79abad7c4c8b3d/src/Command/ErrorFormatter/RawErrorFormatter.php#L22

Reply:
That worked also for the default output. I'm going to add that in as the default in the `composer.json` script.

![verbose](https://github.com/user-attachments/assets/29ae7362-078d-48ab-abe0-f386769c0de6)